handler.ts

```diff
@@ -75,17 +80,22 @@ export const makePeliasRequests = async (
   // to URL parameters before being converted to a string
   const query = new URLSearchParams(event.queryStringParameters).toString()
 
+  // Select which primary geocoder to use.
+  const primaryGeocoder =
```

Comment:
`fetchHere` and `fetchPelias` seem to take arguments in differently. Would it be possible to refactor this to a function that takes one set of args, and then replace all `fetchHere` and `fetchPelias` calls with a call to this new method?